woocommerce/woocommerce-android
# Collect payment button while creating a new order navigates to the Order list instead of Select payment method

Closes: #10773 


### Description

This PR addresses an issue where clicking the 'Collect Payment' button during the creation of a new order incorrectly redirected users to the Order List. With this fix, users will now be properly navigated to the Select Payment Method screen, streamlining the payment collection process for both phone and tablet devices.

### Testing instructions

1. Create new order, add product or custom amount
2. Tap Collect Payment
3. **Ensure you see payment options** - Test the behaviour on both tablet and on phone

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/1331230/67282ce2-3fd2-4ceb-85cc-b6d9398f0809



https://github.com/woocommerce/woocommerce-android/assets/1331230/987acbbc-325d-42c7-867e-06a0ad45eb50
